.github/workflows/quality.yml

```diff
@@ -0,0 +1,28 @@
+name: Quiet Quality
+
+on: push
+permissions: write-all
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Show the merge-base
+        run: git merge-base origin/main ${{ github.sha }}
+        if: github.branch != 'main'
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+          rubygems: latest
+          bundler: none
+
+      - name: Run QuietQuality
+        run: |
+          gem install quiet_quality rubocop standardrb
```

Comment:
I'd rather just use the bundle setup - we're not committing the gemlock, and the constraints are loose, so we'd get to use caching, and it seems a bit simpler.

Reply:
There's a problem with that route. QQ only supports ruby 2.7+, whereas environment_helpers supports 2.6. Adding it to the bundle forces us bump to 2.7. I can leave the _other_ tools being installed from the gemfile, though.

Reply:
Ah! Ugh - I'd definitely want the tool _installed_ in the gemfile, so we don't have to manage it locally as well, but I'd forgotten we supported 2.6 in EH.

I'll give it some thought later, your approach works for now.